.golangci.yml

```diff
@@ -45,3 +50,27 @@ linters-settings:
   revive:
     rules:
       - name: comment-spacings
+  custom:
+    kal:
+      type: "module"
+      description: KAL is the Kube-API-Linter and lints Kube like APIs based on API conventions and best practices.
+      settings:
+        linters:
+          enable:
+          - "commentstart"
+          - "conditions"
+          - "integers"
+          - "jsontags"
+          # NOTE: maxlength check currently trips over our Status fields
+          # missing a MaxLength marker
+          # - "maxlength"
```

Comment:
We should add these. IIUC they add to the runtime complexity of our CEL validations and are calculated at maximum possible length (4MB I think) if this is omitted. Leaving these out could end up tripping us up elsewhere.